### ISSUE TITLE
[Bug]: Terrible load times

### ISSUE THREAD

**Issue by eshsdhsd** [FreeTubeApp/FreeTube#4820]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

Video Shows a loading screen but never starts, or at times takes takes very long
![image](https://github.com/FreeTubeApp/FreeTube/assets/79751239/9e0a4c16-6bb6-4011-9f1e-4be5794cded6)

### Expected Behavior

Not to happen or if it cant load and fails to fallback to invidious

### Issue Labels

accessibility issue, content not loading, inconsistent behavior

### FreeTube Version

v0.19.2 Beta

### Operating System Version

windows 7

### Installation Method

.exe

### Primary API used

Local API

### Additional Information

If I see it happen I will 

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by eshsdhsd** [comment]
Sometimes It takes 7to18 seconds to load something and is stuck at the loading icon, I have no Idea whats causing this and how do I tell whats going on?
![image](https://github.com/FreeTubeApp/FreeTube/assets/79751239/234f0691-d793-4473-a9c4-592a22aafdc5)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
As we dont support the OS you're using in the latest version of FT and this report is about an older unmaintained version of FT. There is no reason to keep this open. 

You probably need to start looking into https://github.com/vxiiduu/VxKex
